react/features/welcome/components/WelcomePage.native.js

```diff
@@ -10,6 +10,8 @@ import { createDesiredLocalTracks } from '../../base/tracks';
 
 import { AbstractWelcomePage, _mapStateToProps } from './AbstractWelcomePage';
 import LocalVideoTrackUnderlay from './LocalVideoTrackUnderlay';
+import { RecentList } from '../../recent-list';
```

Comment:
style: leave empty line, also, this should go after al ./ imports

Reply:
Fixed, thanks! Also the old content of the file